### PR TITLE
Bring up all neighbor ports in DeviceInterfaceLinks

### PR DESCRIPTION
**Pull request by sumukhatv**
**- Why I did it**
To bring up FPGA ports
**- How I did it**
By setting admin_status to up for the remaining ports which are not part of VLAN
**- How to verify it**
Test verifies it
**- Which release branch to backport (provide reason below if seleted)**

- [ ] 201811
- [ ] 201911
- [ ] 202006

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: src/sonic-config-engine/minigraph.py [hunk 1]
```diff
@@ -1033,6 +1033,11 @@ def parse_xml(filename, platform=None, port_config_file=None, asic_name=None, hw
         if port:
             port['admin_status'] = 'up'
 
+    for port in ports.keys():
```
Comment (prsunny) [inline src/sonic-config-engine/minigraph.py:1036]: Just to confirm, `ports `is only from the `DeviceInterfaceLinks `and not the `EthernetInterfaces `from `DeviceInfo`

Comment (sumukhatv) [reply src/sonic-config-engine/minigraph.py:1036]: `ports` is coming from the function `get_port_config`

File: src/sonic-config-engine/minigraph.py [hunk 2]
```diff
@@ -1033,6 +1033,11 @@ def parse_xml(filename, platform=None, port_config_file=None, asic_name=None, hw
         if port:
             port['admin_status'] = 'up'
 
+    for port in neighbors.keys():
```
Comment (prsunny) [inline src/sonic-config-engine/minigraph.py:1036]: We should have a test to cover this.

**Comment by prsunny** [comment]
retest vsimage please

File: src/sonic-config-engine/tests/test_cfggen.py [hunk 3]
```diff
@@ -235,6 +237,44 @@ def test_minigraph_ethernet_interfaces(self):
         output = self.run_script(argument)
         self.assertEqual(output.strip(), "{'lanes': '33,34,35,36', 'fec': 'rs', 'pfc_asym': 'off', 'mtu': '9100', 'alias': 'fortyGigE0/12', 'admin_status': 'up', 'speed': '100000', 'description': 'Interface description'}")
 
+    def test_minigraph_fpga_interfaces(self):
```
Comment (prsunny) [inline src/sonic-config-engine/tests/test_cfggen.py:240]: could you please rename this?

**Comment by jleveque** [comment]
Retest this please

**Comment by prsunny** [comment]
retest this please

**Comment by lguohan** [comment]
why the pr contains hw-mgmt change? @sumukhatv , can you investigating the build failure?

**Comment by lguohan** [comment]
retest vsimage please

**Comment by sumukhatv** [comment]
retest vsimage please

**Comment by rlhui** [comment]
@sumukhatv , is this associated with any bug fix?  Why was it needed in 201911 branch? Thanks.

**Pull request merged**